examples/basics/scene/image.py

```diff
@@ -20,12 +21,29 @@
 view = canvas.central_widget.add_view()
 
 # Create the image
-img_data = np.random.normal(size=(100, 100, 3), loc=128,
-                            scale=50).astype(np.ubyte)
-image = scene.visuals.Image(img_data, parent=view.scene)
-
+#img_data = np.random.normal(size=(100, 100, 3), loc=128,
+#                            scale=50).astype(np.ubyte)
+img_data = np.zeros(25).reshape((5, 5)).astype(np.float32)
+img_data[1:4, 1::2] = 0.5
+img_data[1::2, 2] = 0.5
+img_data[2, 2] = 1.0
+ipol = 'Nearest'
+image = scene.visuals.Image(img_data, ipol=ipol, parent=view.scene)
```

Comment:
It should probably be `interp`, `interp_method`, or `interpolation`. I think I prefer the last one.